docs/api/app.md

```diff
@@ -506,6 +506,41 @@ gets emitted.
 **Note:** Extra command line arguments might be added by Chromium,
 such as `--original-process-start-time`.
 
+### Event: 'notification-activation' _Windows_
+
+Returns:
+
+* `event` Event
+* `appUserModelId` string - contains App module ID
+* `invokedArgs` string - contains arguments(in case of any) for toast button or data property in case of non-persistent notification
+* `dataCount` integer - contains amount of token pairs present inputData string (not null, default value 0 - which means abcense of inputData)
+* `inputData` string - valid JSON-string which contains result of User input into toast Reply fields. Empty string in case of absence of User input (i.e for case dataCount == 0)
```

Comment:
```suggestion
* `notificationData` string - valid JSON-string which contains result of User input into toast Reply fields. Empty string in case of absence of User input (i.e for case dataCount == 0)
```

Reply:
event will be removed

Reply:
I've changed event prototype and name of parameters https://github.com/electron/electron/commit/de1f5b59c626d2be2213c25d33ce105ddcdfa5e6